src/api/v2/orders.rs

```diff
@@ -192,4 +221,37 @@ mod tests {
     // There shouldn't be any other orders with the given ID.
     assert_eq!(filtered.next(), None);
   }
+
+  #[test(tokio::test)]
+  #[serial]
+  async fn symbol_filter_orders() {
+    let api_info = ApiInfo::from_env().unwrap();
+      let client = Client::new(api_info);
+      let order = order_aapl(&client).await.expect("Could not buy appl");
+
+      let request = OrdersReq {
+        symbols: vec!["goog".to_string()],
+        ..Default::default()
+      };
+      let orders = client.issue::<Get>(&request).await.unwrap();
+      assert_eq!(orders.len(), 0);
+
+      let request = OrdersReq {
+        symbols: vec!["aapl".to_string()],
+        ..Default::default()
+      };
+      let orders = client.issue::<Get>(&request).await.unwrap();
+      assert_eq!(orders.len(), 1);
+      assert_eq!(orders[0].symbol, "AAPL");
+
+      // Confirm no symbols correctly returns all open.
+      let request = OrdersReq {
+        ..Default::default()
+      };
+      let orders = client.issue::<Get>(&request).await.unwrap();
+      assert_eq!(orders.len(), 1);
+      assert_eq!(orders[0].symbol, "AAPL");
+
+      cancel_order(&client, order.id).await;
```

Comment:
We should be careful not to leak state should an assertion fail. I think it's easy to rearrange as follows:
```
let order = order_aapl().await.unwrap();
...
let result_goog = client.issue::<Get>(&request_goog).await;
let result_aapl = client.issue::<Get>(&request_aapl).await;
let result_all = client.issue::<Get>(&request_all).await;
cancel_order(&client, order.id).await;

let orders_aapl = result_aapl.unwrap();
// ...
assert_eq!(orders[0].symbol, "AAPL");
```

Reply:
Done.